spec/numberformat.html

```diff
@@ -81,37 +81,46 @@ <h1>InitializeNumberFormat ( _numberFormat_, _locales_, _options_ )</h1>
         1. Set _numberFormat_.[[Notation]] to _notation_.
         1. Perform ? SetNumberFormatDigitOptions(_numberFormat_, _options_, _mnfdDefault_, _mxfdDefault_, _notation_).
         1. Let _compactDisplay_ be ? GetOption(_options_, *"compactDisplay"*, ~string~, &laquo; *"short"*, *"long"* &raquo;, *"short"*).
+        1. Let _defaultUseGrouping_ be *"auto"*.
         1. If _notation_ is *"compact"*, then
           1. Set _numberFormat_.[[CompactDisplay]] to _compactDisplay_.
-        1. Let _useGrouping_ be ? GetOption(_options_, *"useGrouping"*, ~boolean~, ~empty~, *true*).
+          1. Set _defaultUseGrouping_ to *"min2"*.
+        1. NOTE: For historical reasons, the strings *"true"* and *"false"* are accepted and replaced with the default value.
+        1. Let _useGrouping_ be ? GetBooleanOrStringNumberFormatOption(_options_, *"useGrouping"*, &laquo; *"min2"*, *"auto"*, *"always"*, *"true"*, *"false"* &raquo;, _defaultUseGrouping_).
+        1. If _useGrouping_ is *"true"* or _useGrouping_ is *"false"*, set _useGrouping_ to _defaultUseGrouping_.
+        1. If _useGrouping_ is *true*, set _useGrouping_ to *"always"*.
         1. Set _numberFormat_.[[UseGrouping]] to _useGrouping_.
-        1. Let _signDisplay_ be ? GetOption(_options_, *"signDisplay"*, ~string~, &laquo; *"auto"*, *"never"*, *"always"*, *"exceptZero"* &raquo;, *"auto"*).
+        1. Let _signDisplay_ be ? GetOption(_options_, *"signDisplay"*, ~string~, &laquo; *"auto"*, *"never"*, *"always"*, *"exceptZero"*, *"negative"* &raquo;, *"auto"*).
         1. Set _numberFormat_.[[SignDisplay]] to _signDisplay_.
         1. Return _numberFormat_.
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-setnfdigitoptions" type="abstract operation">
-      <h1>
-        SetNumberFormatDigitOptions (
-          _intlObj_: an Object,
-          _options_: an Object,
-          _mnfdDefault_: a Number,
-          _mxfdDefault_: a Number,
-          _notation_: a String,
-        )
-      </h1>
-      <dl class="header">
-        <dt>description</dt>
-        <dd>It populates the internal slots of _intlObj_ that affect locale-independent number rounding (see <emu-xref href="#sec-formatnumberstring"></emu-xref>).</dd>
-      </dl>
+    <emu-clause id="sec-setnfdigitoptions" aoid="SetNumberFormatDigitOptions">
+      <h1>SetNumberFormatDigitOptions ( _intlObj_, _options_, _mnfdDefault_, _mxfdDefault_, _notation_ )</h1>
```

Comment:
Is there an important reason to move away from the structured headers syntax here? Since the new description doesn't include the "type" information unlike the previous style, we're just losing that information...

Reply:
May have gotten lost as part of the merge. Will fix.